hastexo/hastexo.py

```diff
@@ -219,12 +219,13 @@ def parse_attributes(tag, node, block):
                 # This will raise a TypeError if the string literal
                 # cannot be converted
                 capacity = int(capacity)
-            template = node.attrib.get("template", None)
             environment = node.attrib.get("environment", None)
             provider = {"name": name,
                         "capacity": capacity,
-                        "template": template,
                         "environment": environment}
+            template = node.attrib.get("template", None)
```

Comment:
Let's please also apply the same logic to `environment` and `capacity`.

Reply:
Ack, sure.